services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -164,6 +164,18 @@ class LinkPropertiesSpec extends BaseSpecification {
         islUtils.getIslInfo(updatedLinks, isl).get().maxBandwidth == maxBandwidthValue.toInteger()
         islUtils.getIslInfo(updatedLinks, isl.reversed).get().maxBandwidth == maxBandwidthValue.toInteger()
 
+        when: "Update cost on forward direction of the link props"
+        def newCostValue = "345"
+        def newLinkPropsInfo = northbound.getAllLinkProps().first().tap { it.props.cost = newCostValue }
+        northbound.updateLinkProps([newLinkPropsInfo])
```

Comment:
As far as I understand you can do something like that:
`northbound.updateLinkProps([islUtils.toLinkProps(isl, ["cost": newCostValue])])` and remove L169.

Reply:
updated